pipeline-examples/parallel-multiple-nodes/ParallelMultipleNodes.groovy

```diff
@@ -0,0 +1,12 @@
+def labels = ['precise', 'trusty'] // labels for Jenkins node types we will build on
+def builders = [:]
+for (x in labels) {
```

Comment:
When I tried it the other way, I got two parallel builds on the same label. I'll add a comment.